scenarios/appsec/test_ip_blocking.py

```diff
@@ -0,0 +1,74 @@
+# Unless explicitly stated otherwise all files in this repository are licensed under the the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2021 Datadog, Inc.
+import json
+
+from scenarios.remote_config.test_remote_configuration import rc_check_request
+from utils import BaseTestCase, context, coverage, interfaces, released, rfc, bug
+from utils.tools import logger
+
+with open("scenarios/appsec/rc_expected_requests_asm_data.json", encoding="utf-8") as f:
+    EXPECTED_REQUESTS = json.load(f)
+
+
+@rfc("https://docs.google.com/document/d/1GUd8p7HBp9gP0a6PZmDY26dpGrS1Ztef9OYdbK3Vq3M/edit")
+@released(cpp="?", dotnet="2.16.0", java="0.110.0", php="?", python="?", ruby="?", nodejs="?", golang="?")
+@coverage.basic
+class Test_AppSecIPBlocking(BaseTestCase):
```

Comment:
IIUC, we can't really change the RC answers during a test for example to "suddenly" change the list of blocked IPs in the middle of a test, right?

Reply:
Yes you can, but I have to do some work for that :)

Reply:
ok, really great news, because this is where we will most likely find bugs 😬
thanks a lot for your work on this - a new era for our appsec system tests beings ⚔️ 😆